Update Dfsu documentation (#349)
DHI/mikeio: docs/dfsu-2dv-vertical-profile.md

# Dfsu 2DV Vertical Profile

> Apart from the common [dfsu-geometry properties and methods](./dfu-mesh-overview.md#mike-io-flexible-mesh-geometry), Dfsu2DV has the below additional *properties* (from it's geometry [GeometryFMVerticalProfile](GeometryFMVerticalProfile)): 

Reviewer: ```suggestion
In addition to the common [dfsu-geometry properties and methods](./dfu-mesh-overview.md#mike-io-flexible-mesh-geometry), Dfsu2DV has the below additional *properties* (from it's geometry [GeometryFMVerticalProfile](GeometryFMVerticalProfile)): 
```

```{eval-rst}
.. autosummary::
    :nosignatures:

    mikeio.dfsu_layered.Dfsu2DV.n_layers
    mikeio.dfsu_layered.Dfsu2DV.n_sigma_layers
    mikeio.dfsu_layered.Dfsu2DV.n_z_layers
    mikeio.dfsu_layered.Dfsu2DV.layer_ids
    mikeio.dfsu_layered.Dfsu2DV.top_elements
    mikeio.dfsu_layered.Dfsu2DV.bottom_elements
    mikeio.dfsu_layered.Dfsu2DV.n_layers_per_column
    mikeio.dfsu_layered.Dfsu2DV.e2_e3_table
    mikeio.dfsu_layered.Dfsu2DV.elem2d_ids
```

> And apart from the basic dfsu functionality, Dfsu2DV has the below additional *methods*: 

Reviewer: ```suggestion
And in addition to the basic dfsu functionality, Dfsu2DV has the below additional *methods*: 
```
